Source/Core/Common/CommonPaths.h

```diff
@@ -235,4 +235,9 @@ const std::string& GetDumpSSLDir()
 {
   return GetUserPath(D_DUMPSSL_IDX);
 }
+
+const std::string& GetLoadDir()
```

Comment:
Not sure if "Load" is a good name (which also goes for the IDX tho), it doesn't really make sense to me as a term - especially not since it's only used for `titles.txt` (which _is_ being loaded, but thats probably true for half the other files too)

I don't think I can come up with a better name though; so unless someone else has a good idea you can ignore this comment...